### ISSUE TITLE
Feature: Add Dockerfile for use

### ISSUE THREAD

**Issue by t3hmrman** [VADOSWARE/pg_idkit#8]
# Context

The `Dockerfile` that works for building this 

# Proposal

Add appropriate `Dockerfile`s for to make building and using `pg_idkit` easier.

Make use of the [exploratory work already done](https://github.com/VADOSWARE/pg_idkit/issues/4#issuecomment-1248882188).

Once this feature is complete, we should have `Dockerfile`s for at least Alpine and Debian based distros.

- [ ] Write `Dockerfile` for Alpine Linux
- [ ] Write `Dockerfile` for Debian distro
- [ ] Update documentation (`README.md` at least)

# Alternatives

Leave the functions as they are.

# Additional Links

N/A

**Comment by t3hmrman** [comment]
Note that we need Dockerfiles across *multiple* matrices:

- Postgres version
- pg_idkit version (current is fine)
- debian vs alpine linux (this is optional, we can start with only one at first)

To get things moving quickly I think it's fine to build this as simply as possible at first (different `Dockerfile` for every combination -- ex. `pgidkit-pg15.4-alpine3.18.Dockerfile`), and then optimize/consolidate later.
